demo/svelte.config.js

```diff
@@ -50,6 +50,7 @@ const config = {
 
 		version: {
 			name: child_process.execSync('git rev-parse HEAD').toString().trim(),
+			pollInterval: 5_000,
```

Comment:
Do we really need to poll each 5 sec for an app version? Did you see any performance issue with this option?
It can be useful for devs but I don't think we will release latest so many times to use this option. But if it is negligible the drawback, why not :)